gui.js

```diff
@@ -1552,9 +1552,20 @@ IDE_Morph.prototype.fixLayout = function (situation) {
 IDE_Morph.prototype.setProjectName = function (string) {
     this.projectName = string.replace(/['"]/g, ''); // filter quotation marks
     this.hasChangedMedia = true;
+    this.updateDocumentTitleText();
```

Comment:
Okay. Where else would we need to change the title other than when we're opening a project?